### PR TITLE
Post list name refactoring

### PR DESCRIPTION
**Pull request by malinajirka**
Follow-up to [this](https://github.com/wordpress-mobile/WordPress-Android/pull/9491#discussion_r270838712) discussion.

Minor refactoring of names in PostListViewHolder and related classes.

To test:
1. Open the post list
2. Make sure statuses label is being correctly displayed (eg. when you make changes to a post while offline, yellow "Local changes" status is shown)

### CODE REVIEW

File: WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt [hunk 1]
```diff
@@ -31,13 +31,13 @@ class PostListItemViewHolder(
...
-    private val tvExcerpt: WPTextView = itemView.findViewById(R.id.excerpt)
-    private val tvDateAndAuthor: WPTextView = itemView.findViewById(R.id.date_and_author)
-    private val tvStatusLabels: WPTextView = itemView.findViewById(R.id.status_labels)
-    private val pbProgress: ProgressBar = itemView.findViewById(R.id.progress)
-    private val flDisabledOverlay: FrameLayout = itemView.findViewById(R.id.disabled_overlay)
+    private val featuredImageView: ImageView = itemView.findViewById(R.id.image_featured)
+    private val titleTextView: WPTextView = itemView.findViewById(R.id.title)
+    private val excerptTextView: WPTextView = itemView.findViewById(R.id.excerpt)
+    private val dateAndAuthorTextView: WPTextView = itemView.findViewById(R.id.date_and_author)
+    private val statusesLabelTextView: WPTextView = itemView.findViewById(R.id.statuses_label)
```
Comment (shiki) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt]: 👍

File: WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt [hunk 2]
```diff
@@ -31,13 +31,13 @@ class PostListItemViewHolder(
...
+    private val progressProgressBar: ProgressBar = itemView.findViewById(R.id.progress)
```
Comment (shiki) [inline WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt]: ℹ️ This seems a bit generic. Does `uploadProgressBar` sound and explain its purpose better? 🤔

File: WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiState.kt [hunk 3]
```diff
@@ -17,9 +17,9 @@ data class PostListItemUiStateData(
     val excerpt: UiString?,
     val imageUrl: String?,
     val dateAndAuthor: UiString?,
-    @ColorRes val statusLabelsColor: Int?,
-    val statusLabels: List<UiString>,
-    val statusLabelsDelimiter: UiString,
+    @ColorRes val statusesColor: Int?,
```
Comment (shiki) [inline WordPress/src/main/java/org/wordpress/android/viewmodel/posts/PostListItemUiState.kt]: 👍

File: WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt [hunk 4]
```diff
@@ -46,26 +46,26 @@ class PostListItemViewHolder(
...
-        updateStatusLabels(tvStatusLabels, item.data.statusLabels, item.data.statusLabelsDelimiter)
-        if (item.data.statusLabelsColor != null) {
-            tvStatusLabels.setTextColor(ContextCompat.getColor(tvStatusLabels.context, item.data.statusLabelsColor))
+        setTextOrHide(titleTextView, item.data.title)
+        setTextOrHide(excerptTextView, item.data.excerpt)
+        setTextOrHide(dateAndAuthorTextView, item.data.dateAndAuthor)
+        uiHelpers.updateVisibility(statusesLabelTextView, item.data.statuses.isNotEmpty())
+        updateStatusesLabel(statusesLabelTextView, item.data.statuses, item.data.statusesDelimiter)
+        if (item.data.statusesColor != null) {
+            statusesLabelTextView.setTextColor(ContextCompat.getColor(statusesLabelTextView.context, item.data.statusesColor))
```
[the remark this answers is not in the document]
Comment (malinajirka) [reply WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt]: Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9514/commits/092f3d73eec6d6aaeba64569f78ea1868dfee643

File: WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt (hunk 2 above)
Comment (malinajirka) [reply WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt]: Fixed in https://github.com/wordpress-mobile/WordPress-Android/pull/9514/commits/092f3d73eec6d6aaeba64569f78ea1868dfee643

**Comment by malinajirka** [comment]
Thanks @shiki ! Sorry I created the PR and left so I haven't noticed the lint issue. I've fixed it + renamed the uploadProgressBar variable.

**Comment by shiki** [comment]
No worries @malinajirka!

**Pull request merged**